### DISCUSSION TITLE
Touchscreen not working for interactive elements on Flash stage but the Ruffle UI

### DISCUSSION THREAD

**Discussion by nphuracm** [ruffle-rs/ruffle#14245] [Q&A]
I have been using Ruffle on a touchscreen device with the standalone player. When playing a Flash with interactive elements on stage (buttons, sliders, things like that), touching the elements directly with the touchscreen will not work, and I have to use a cursor to interact with these.

**Comment by Lord-McSweeney** [comment]
I opened an issue for this: #14246
